style.css

```diff
@@ -0,0 +1,79 @@
+body {
+  font-family: "poppins", sans-serif;
+  background-image: url(assets/header-shapesmobile.png);
+}
+
+nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 10px;
+}
+
+span {
+  color: #7f8cff;
+  font-weight: 700px;
+  font-size: 18px;
+  line-height: 20px;
+}
+
+main {
+  display: flex;
+  margin: 20px;
+  justify-content: center;
+}
+
+h2 {
+  font-size: 50px;
+  line-height: 52px;
+}
+
+article {
+  font-weight: 400px;
+  font-size: 16 px;
+  line-height: 24px;
+}
+
+h4 {
+  color: #7f8cff;
+  margin: 0;
+}
+
+nav > h4 {
+  font-size: 20px;
+  line-height: 28px;
+}
+
+main > div > h4 {
+  margin-top: 20px;
+}
```

Comment:
I kindly suggest you to stick to Figma design, currently your design and Figma design are different:

Your design : 
- the background image is cut on the right, please correct it using `background-position` or `background-size` tag
- the place of the element is not like the design, you can correct it using `flex-grow flex-shrink` or using `margin and padding`.

<img width="448" alt="image" src="https://user-images.githubusercontent.com/7243132/170326545-931bed2b-ee64-4370-ab52-281bfb6a923d.png">

Figma : 
<img width="249" alt="image" src="https://user-images.githubusercontent.com/7243132/170326739-f1b74051-8213-4427-9ee3-abc1871489a3.png">